account_payment_order/models/account_move_line.py

```diff
@@ -23,6 +26,25 @@ class AccountMoveLine(models.Model):
         string="Payment lines",
     )
 
+    @api.depends(
+        "move_id", "move_id.invoice_partner_bank_id", "move_id.payment_mode_id"
+    )
+    def _compute_partner_bank_id(self):
+        for ml in self:
+            res = None
```

Comment:
Better to avoid possible side effects using a falsy but not expected value

```suggestion
            res = False
```